eng/Version.Details.xml

```diff
@@ -1,13 +1,13 @@
 <Dependencies>
   <ProductDependencies></ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.19113.1">
```

Comment:
Shouldn't there be a corresponding update to `eng/Versions.props` and one that updates `MicrosoftNETCorePlatformsVersion` and `MicrosoftPrivateCoreFxNETCoreAppVersion` (at a minimum)?